apps/judicial-system/web/src/components/FormFooter/FormFooter.tsx

```diff
@@ -24,37 +29,49 @@ interface Props {
 
 const FormFooter: React.FC<Props> = (props: Props) => {
   const router = useRouter()
+  const { width } = useWindowSize()
+  const isMobile = width <= theme.breakpoints.md
 
   return (
     <Box display="flex" justifyContent="spaceBetween" alignItems="center">
-      <Button
-        variant="ghost"
-        disabled={props.previousIsDisabled}
-        onClick={() => {
-          router.push(props.previousUrl ?? '')
-        }}
-      >
-        Til baka
-      </Button>
-      {!props.hideNextButton && (
+      <Box className={styles.footerItem}>
         <Button
-          data-testid="continueButton"
-          icon={props.nextButtonIcon ?? 'arrowForward'}
-          disabled={props.nextIsDisabled}
-          colorScheme={props.nextButtonColorScheme ?? 'default'}
-          loading={props.nextIsLoading}
+          variant="ghost"
+          disabled={props.previousIsDisabled}
           onClick={() => {
-            if (props.onNextButtonClick) {
-              props.onNextButtonClick()
-            } else if (props.nextUrl) {
-              router.push(props.nextUrl)
-            }
+            router.push(props.previousUrl ?? '')
           }}
+          icon={isMobile ? 'arrowBack' : undefined}
+          circle={isMobile}
         >
-          {props.nextButtonText ?? 'Halda áfram'}
+          {!isMobile && 'Til baka'}
```

Comment:
Please consider adding `aria-label` for moblie since we are not displaying the label